src/main/java/me/desht/pneumaticcraft/common/block/entity/EtchingTankBlockEntity.java

```diff
@@ -328,8 +328,8 @@ private class EtchingFluidTank extends SmartSyncTank {
         }
 
         @Override
-        public boolean isFluidValid(FluidStack stack) {
-            return stack.getFluid().is(PneumaticCraftTags.Fluids.ETCHING_ACID);
+        public boolean isFluidValid(FluidStack fluidStack) {
```

Comment:
Why this change?  What's wrong with checking for the fluid tag here?

Reply:
if I check the tag it means custom empty PCBs all have to use the same fluids for etching, for example you might want a custom PCB that needs it's own etching fluid to etch rather than using the vanilla pneumaticcraft one